website-phishing/index.js

```diff
@@ -21,9 +21,79 @@ import {WebsitePhishingDataset} from './data';
 import * as ui from './ui';
 import * as utils from './utils';
 
+function falsePositives(yTrue, yPred) {
```

Comment:
This is available in Layers API but hasn't been publically exposed. Since they are available in [TensorFlow Python API](https://www.tensorflow.org/api_docs/python/tf/metrics/false_positives), I feel we should make them public and use `tf.metrics` here. I can raise a PR for it.